### DISCUSSION TITLE
Results grouping by repository

### DISCUSSION THREAD

**Discussion by marik22312** [community/community#37873] [Other Feature Feedback, Questions, & Ideas]
Hey all!
started to use Code Search lately, tried to search for all usages of a specific string in our company's codebase, but unfortunately got pretty confused and got duplicated results. after a closer look, it seems that Code Search doesn't group the files by their repo, making some files that belong to the same repository - to appear on different pages

is there a plan to support repository grouping?
i.e all the results for a certain repository will be under the same section?

**Comment by ossanna16** [comment]
Hi there @marik22312 and welcome to our community! Thank you for your feature request!
